server/hedley/modules/custom/hedley_fields/hedley_fields.features.field_base.inc

```diff
@@ -963,12 +963,13 @@ function hedley_fields_field_default_field_bases() {
     'module' => 'list',
     'settings' => array(
       'allowed_values' => array(
-        'add to calendar icon' => 'add to calendar icon',
-        'alarm icon' => 'alarm icon',
-        'bug icon' => 'bug icon',
-        'paw icon' => 'paw icon',
-        'find icon' => 'find icon',
-        'handshake icon' => 'handshake icon',
+        'calendar' => 'Add to calendar',
```

Comment:
actually, lets change it. Can you please open a follow up issue for @nader77 (1h?) to deal with that. Wrong data is a definitely going to bite us in the future.